src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -640,28 +642,28 @@ private int HandleResponse(BuildResponse response, string pathToTool, string res
                     return completedResponse.ReturnCode;
 
                 case BuildResponse.ResponseType.MismatchedVersion:
-                    LogError("Roslyn compiler server reports different protocol version than build task.");
+                    logError("Roslyn compiler server reports different protocol version than build task.");
                     return base.ExecuteTool(pathToTool, responseFileCommands, commandLineCommands);
 
                 case BuildResponse.ResponseType.IncorrectHash:
-                    LogError("Roslyn compiler server reports different hash version than build task.");
+                    logError("Roslyn compiler server reports different hash version than build task.");
                     return base.ExecuteTool(pathToTool, responseFileCommands, commandLineCommands);
 
                 case BuildResponse.ResponseType.Rejected:
                 case BuildResponse.ResponseType.AnalyzerInconsistency:
-                    CompilerServerLogger.LogError($"Server rejected request {response.Type}");
+                    logger.LogError($"Server rejected request {response.Type}");
```

Comment:
Should these be using the params version, to avoid realizing unneeded strings?

Reply:
Commented on this a bit above: the strings have virtually no impact on the performance trace here hence I error'd on usability here. Don't feel particularly strongly abuot it though.